tools/serverpod_cli/lib/src/generator/open_api/open_api_definition.dart

```diff
@@ -0,0 +1,200 @@
+library openapi_definition;
+
+import 'package:recase/recase.dart';
+
+import '../../../analyzer.dart';
```

Comment:
Typically, prefer package paths in imports. There may be places where this isn't an option, but as a general rule. :)